### ISSUE TITLE
Support email addresses with +

### ISSUE THREAD

**Issue by pas256** [danilop/LambdAuth#12]
Hi there,

I just tried the sample application you have (thanks for that) and I signed up using an email address that contains a plus. The format being: `me+lambdauth@example.com`

I recieved the verification email, clicked the link, but the verification failed. I am guessing it is the plus character because it was not URL encoded in the verification link.

**Comment by pas256** [comment]
The error message also did not contain the plus character

```
User me lambdauth@example.com has not been Verified, sorry.
```

**Comment by danilop** [comment]
I think you're right, I just put a quick fix in the master branch, using encodeURIComponent for the email in the verification link. Do you have time to check if that works through the verify.html page?

**Comment by pas256** [comment]
Testing here: http://lambdauth.danilop.net/

Still fails

**Comment by danilop** [comment]
Yes - I didn't redeploy there yet - I'll let you know, sorry for the
misunderstanding.

On Sat, Aug 29, 2015 at 12:46 AM, Peter Sankauskas <notifications@github.com
